packages/kbn-kibana-manifest-schema/src/kibana_json_v2_schema.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { JSONSchema } from 'json-schema-typed';
+import { desc } from './desc';
+
+export const PLUGIN_ID_PATTERN = /^[a-z][a-zA-Z_]*$/;
+
+export const MANIFEST_V2: JSONSchema = {
```

Comment:
Should we think about also introducing a version field for this updated `kibana.jsonc` (v2) to accommodate for new additions or changes in the future?

Reply:
I'd really like to avoid needing a version field at this point but I explicitly didn't use version for the "package version" like v1 did so if we do need it in the future, and parsing all the manifests to understand their version isn't a problem, then we can add a `version` field and assume undeclared versions are `v1`.